kahuna/public/stylesheets/main.css

```diff
@@ -1046,21 +1045,31 @@ FIXME: what to do with touch devices
     margin-right: 300px;
 }
 
-.image-info {
-    margin-bottom: 15px;
+.image-info__group {
+    padding: 10px;
+    border-bottom: 1px solid #565656;
+} 
+
+.image-info:last-child .image-info__group{
+    border-bottom: none;
+}   
+
+.image-info__group table {
```

Comment:
Any chance we could give these tables classnames, so if we do move to `dl` or whatever we can just use those?